CODING.md

```diff
@@ -0,0 +1,36 @@
+# Coding Guidelines
+
+## General Notes
+In general we aim for keeping the coding guidelines that are common by the Java Language. This includes code style,
+as well as the usage of coding patterns. Exceptions have to get discusses and agreed with the project leads.
+
+## Checkstyle Config
+This repository contains a `checkstyl.xml` Checkstyle config which gets automatically used by the CI Runner for code 
+validation on the **master**, **develop** and **Pull Requests** branches.
+
+Feature branches do not checked because we want to things getting done quickly. This can be done if everybody codes as 
```

Comment:
```suggestion
Feature branches are not automatically checked because we want to get development done quickly. To achieve this, everybody may codes as 
```